### PR TITLE
Moving SAPI to standalone package

### PR DESCRIPTION
**Pull request by msmakouz**
| Q             | A
| ------------- | ---
| Bugfix?       | ❌
| Breaks BC?    | ✔️
| New feature?  | ❌

### CODE REVIEW

File: src/Framework/Debug/StateCollector/EnvironmentCollector.php [hunk 1]
```diff
@@ -23,11 +22,8 @@ public function populate(StateInterface $state): void
     {
         $state->setTag('php', \phpversion());
 
-        if (
-            $this->container->has(DispatcherInterface::class) &&
-            $this->container->get(DispatcherInterface::class) instanceof SapiDispatcher
-        ) {
-            $state->setTag('dispatcher', 'sapi');
+        if ($this->container->has(DispatcherInterface::class)) {
+            $state->setTag('dispatcher', DispatcherInterface::class);
```
Comment (roxblnfk) [inline src/Framework/Debug/StateCollector/EnvironmentCollector.php:26]: I am not sure about this change yet

**Review by roxblnfk — COMMENTED** [review]
It need to remove this line from changelog:
```
- [spiral/http] Added return type `bool` to the method `emit` in `Spiral\Http\EmitterInterface` interface.
```

File: src/Framework/Debug/StateCollector/EnvironmentCollector.php (hunk 1 above)
Comment (roxblnfk) [reply src/Framework/Debug/StateCollector/EnvironmentCollector.php:26]: What about 
```suggestion
            $state->setTag('dispatcher', $this->container->get(DispatcherInterface::class)::class);
```

Comment (msmakouz) [reply src/Framework/Debug/StateCollector/EnvironmentCollector.php]: Removed:
- [spiral/http] Added return type `bool` to the method `emit` in `Spiral\Http\EmitterInterface` interface.

Changed `$state->setTag('dispatcher', DispatcherInterface::class);` to `$state->setTag('dispatcher', $this->container->get(DispatcherInterface::class)::class);`

**Pull request merged**